Cargo.toml

```diff
@@ -27,6 +27,10 @@ gccjit = { git = "https://github.com/antoyo/gccjit.rs" }
 # Local copy.
 # gccjit = { path = "../gccjit.rs" }
 
+object = { version = "0.30.1", default-features = false, features = [
+    "std",
+    "read",
```

Comment:
You may want to use `read_core` + the individual file formats + `unaligned` instead to avoid a dependency on `wasmparser`.

Reply:
Are you sure? It looks like the `wasm` stuff is not enabled by `read`.

Reply:
You're right. My bad.